src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -32,27 +34,48 @@ export type HorizontalTabsProps = {
    * @default 'default'
    */
   type?: 'default' | 'borderless' | 'pill';
+  /**
+   * Modify height of tabs. Use 'large' when tabs have sublabels.
+   * @default 'default'
+   */
+  size?: 'default' | 'large';
```

Comment:
ahh.. This one..would be great if it works same as we have done with Menu..

Reply:
cant. Size is controlled in tabs, sublabel is controlled in individual tab

Reply:
I know, thats what I am concerned about now..